zammad/templates/configmap-nginx.yaml

```diff
@@ -42,6 +42,9 @@ data:
             proxy_set_header Connection "Upgrade";
             proxy_set_header CLIENT_IP $remote_addr;
             proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+            {{ if .Values.envConfig.zammad.nginx.websocketExtraHeaders }}
+                {{ .Values.envConfig.zammad.nginx.websocketExtraHeaders | nindent 12 }}
```

Comment:
Imho this should look like:

```
            proxy_set_header  {{ .Values.envConfig.zammad.nginx.websocketExtraHeaders }}
```

Reply:
I need to modify more than a single header so that will not work for my intended use case. Would you prefer I make the value a list and iterate over the items?